src/cba-list-new/shadow.css

```diff
@@ -149,7 +149,6 @@ ul li
 
 li > .row.highlight
 {
-  background-color: var(--color-selected);
```

Comment:
As [was mentioned here](https://github.com/browser-automation/cba-components/pull/63#discussion_r642527161) you might want to add background style for selected item.

Reply:
Sorry, I don't get it. You mean the bgrnd color of the hovered row? It is now - light grey